projects/analogue-paths/index.md

```diff
@@ -0,0 +1,70 @@
+--- 
+authors: 
+  - Joshua Pacheco
+authors-url: 
+  - "http://incom.org/profil/7029"
+layout: project
+title: "Paths: An exploration of analogue algorithms"
+type: project
+---
+
+# PATHS
```

Comment:
Please start at H2 the H1 Headline will be added by the system https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax